### PR TITLE
Court probation prod certificates route53

### PR DESCRIPTION
**Pull request by theDustRoom**

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    is-production          = var.is-production
+    environment-name       = var.environment-name
+    owner                  = var.team_name
+    infrastructure-support = var.infrastructure-support
+  }
+}
+
+resource "kubernetes_secret" "route53_zone_sec" {
+  metadata {
+    name      = "route53-zone-output"
```
Comment (mtrbls) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf:16]: could you make sure the secrets are named differently ?

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf [hunk 2]
```diff
@@ -0,0 +1,23 @@
+resource "aws_route53_zone" "route53_zone" {
```
Comment (mtrbls) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf:1]: could you make sure both route53 zones are named differently please ?

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    business-unit          = var.business-unit
+    application            = var.application
+    is-production          = var.is-production
+    environment-name       = var.environment-name
+    owner                  = var.team_name
+    infrastructure-support = var.infrastructure-support
+  }
+}
+
+resource "kubernetes_secret" "route53_zone_sec" {
```
Comment (mtrbls) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf:14]: Could you make sure both route53 secret are named differently please ?

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf (hunk 1 above)
Comment (mtrbls) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf:16]: Same here please

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf (hunk 2 above)
Comment (theDustRoom) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf]: Done

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf (hunk 3 above)
Comment (theDustRoom) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf]: Done

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf (hunk 1 above)
Comment (theDustRoom) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf:16]: Does the name here also have to be unique?

Comment (mtrbls) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf:16]: yes, it does. 
Also, for clarity it is better to have easily identifiable secrets name.

Comment (theDustRoom) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf]: OK, thanks. Updated now.

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/crime-portal-mirror-gateway-route53.tf [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+resource "kubernetes_secret" "crime_portal_mirror_gateway_route53_zone_sec" {
+  metadata {
+    name      = "crime-portal-mirror-gateway-route53-zone-output"
+    namespace = var.namespace
+  }
+
+  data = {
+    zone_id = aws_route53_zone.route53_zone.zone_id
```
Comment (mtrbls) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/crime-portal-mirror-gateway-route53.tf:21]: `aws_route53_zone.route53_zone.zone_id` should be `aws_route53_zone.crime_portal_mirror_gateway_route53_zone.zone_id` here

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+resource "kubernetes_secret" "prepare_case_route53_zone_sec" {
+  metadata {
+    name      = "prepare-case-route53-zone-output"
+    namespace = var.namespace
+  }
+
+  data = {
+    zone_id = aws_route53_zone.route53_zone.zone_id
```
Comment (mtrbls) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf:21]: Please amend this to match the route53 resource name, as described in the above comment

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/crime-portal-mirror-gateway-route53.tf (hunk 4 above)
Comment (theDustRoom) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/crime-portal-mirror-gateway-route53.tf]: Thanks for the response, this has now been changed

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf (hunk 5 above)
Comment (theDustRoom) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/prepare-case-route53.tf]: This has now been changed

**Pull request merged**